K2Bridge/KustoDAL/KustoResponseParser.cs

```diff
@@ -173,9 +174,13 @@ private void ReportQueryExecutionMetrics(KustoResponseDataSet kustoResponseDataS
                 response.AddParentToAgg(parent);
 
                 // Determine how to create the buckets based on the aggregation type
-                Func<DataRow, IBucket> createBucketFromDataRow = null;
+                Func<DataRow, Bucket> createBucketFromDataRow = null;
```

Comment:
Do you know why we used IBucket in the first place? Now that you changed it, it doesn't seem to be used anywhere. So either delete it or find why we need it

Reply:
`IBucket` was an interface with a single property in it. After the first review pass it did not seem useful to have an interface for this and I switched to a base class. But I forgot to delete the interface source files, this is done now.